### ISSUE TITLE
Expo SDK 50 Build Error

### ISSUE THREAD

**Issue by bendrobinson1998** [klaviyo/klaviyo-react-native-sdk#108]
### Description

```markdown
I am getting this error on build:

  1 | #import "KlaviyoReactNativeSdk.h"
> 2 | #import <klaviyo_react_native_sdk-Swift.h>
    |         ^ 'klaviyo_react_native_sdk-Swift.h' file not found
  3 | 
  4 | @implementation KlaviyoReactNativeSdk
  5 | RCT_EXPORT_MODULE()
```

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla project
- [ ] If possible, I've reproduced the issue using the `master` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/klaviyo/klaviyo-react-native-sdk/issues) or [pull request](https://github.com/klaviyo/klaviyo-react-native-sdk/pulls).

**Comment by evan-masseau** [comment]
Hi, while we have not tested against expo explicitly, we've have heard from a few users who saw this error and resolved it by cleaning their build directories, deleting derived data, and running a clean pod install. Based on the information you've provided, this is really all I can guess.

**Comment by bendrobinson1998** [comment]
Guys this still isn't fixed, It seems to be IOS only. Can you please reopen this and re address it? Ive been trying all the suggestions and my environment is as recceomnded.

**Comment by evan-masseau** [comment]
The header import issue was patched in [Version 0.2.1](https://github.com/klaviyo/klaviyo-react-native-sdk/releases/tag/0.2.1).
